### PR TITLE
Full website

### PR DESCRIPTION
**Pull request by mengiefen**
The full website is done according to the given template. The routing is done using `react-router-dom` library.

### CODE REVIEW

File: src/Pages/Qoutes.jsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import React, { useEffect, useState } from 'react';
+
+const Qoutes = () => {
+  const [qoute, setQoute] = useState('');
+  // const [source, setSource] = useState('');
```
Comment (Meltrust) [inline src/Pages/Qoutes.jsx]: - [x] Please, remove this old piece of commented code in order to improve the cleanliness of the repo 💪

File: src/Pages/Qoutes.jsx [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  useEffect(async () => {
+    const index = Math.floor(Math.random() * 15);
+    const response = await fetch('https://dummyjson.com/quotes/random');
+    const result = await response.json();
+    const image = await fetch('https://api.pexels.com/v1/search?query=nature');
+    const imgResult = await image.json();
+    // setSource(imgResult.photos[index].src.large);
```
Comment (Meltrust) [inline src/Pages/Qoutes.jsx]: - [x] Please, remove this old piece of commented code in order to improve the cleanliness of the repo 💪

File: src/Pages/Qoutes.jsx [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+    console.log(imgResult.photos[index].src.large);
+    setQoute(result.quote);
+  }, []);
+
+  // const background = {
+  //   backgroundImage: `${source}`,
+  // };
```
Comment (Meltrust) [inline src/Pages/Qoutes.jsx]: - [x] Please, remove this old piece of commented code in order to improve the cleanliness of the repo 💪

**Review by Meltrust — CHANGES_REQUESTED** [review]
Hi @mengiefen,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



### To highlight:

- Your calculator is incredibly beautiful.  Congratulations.✔️ 

You can do it

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by micheaol — APPROVED** [review]
# STATUS: APPROVED :green_circle: :clap: 

Hi @mengiefen , 

Great work on making the changes requested by the previous reviewer 👏 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## TO HIGHLIGHT: 
- Your project is professional 💯 
- Your repo is professional 💯 
- Well structured files 💯 
- You set up linter files and no errors. 💯 

well-1

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**